core/templates/dev/head/pages/library-page/search-bar/search-bar.directive.html

```diff
@@ -110,6 +108,19 @@
 </div>
 
 <style>
+  .form-control:focus {
+    border-color: #66afe9;
+    box-shadow: inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6); 
+    outline: 0;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);
+}
```

Comment:
Indent this and subsequent lines by 2. Please follow the correct indentation patterns.

Reply:
Done Thanks!